shell/platform/windows/platform_handler.cc

```diff
@@ -227,6 +227,7 @@ PlatformHandler::PlatformHandler(
       return std::make_unique<ScopedClipboard>();
     };
   }
+  FlutterDesktopPluginRegistrarRegisterTopLevelWindowProcDelegate(engine_->GetRegistrar(), PlatformHandler::WindowProcCallback, this);
```

Comment:
I'm not sure I like having Windows messaging handling in `PlatformHandler`. Personally, I'd prefer if `PlatformHandler` focused on just platform channel messaging (receiving and sending messages to the framework).

Could we move this logic to a new class whose top-level window message handler is registered by `FlutterWindowsEngine`?